lib/new_relic/agent/llm_event.rb

```diff
@@ -0,0 +1,39 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'llm_event/chat_completion'
+require_relative 'llm_event/chat_completion/message'
+require_relative 'llm_event/chat_completion/summary'
+require_relative 'llm_event/embedding'
+require_relative 'llm_event/feedback'
+require_relative 'llm_event/response_headers'
+
+module NewRelic
+  module Agent
+    class LlmEvent
+      # response_model looks like repsonse.model
+      INGEST_SOURCE = 'Ruby'
+
+      def initialize(id: nil, request_id: nil, span_id: nil, transaction_id: nil, trace_id: nil, response_model: nil, vendor: nil, **args)
```

Comment:
Is the double splat intended to catch something in the future? If not, I'm thinking it can be removed or perhaps the method input can be changed to simply be a hash instead of named arguments.